tests/Jellyfin.Api.Tests/Controllers/EncodedQueryStringTest.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Jellyfin.Api.Tests.Controllers
+{
+    /// <summary>
+    /// Defines the test for encoded querystrings in the url.
+    /// </summary>
+    public class EncodedQueryStringTest : IClassFixture<JellyfinApplicationFactory>
+    {
+        private readonly JellyfinApplicationFactory _factory;
+
+        public EncodedQueryStringTest(JellyfinApplicationFactory factory)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async Task Ensure_Ping_Working()
+        {
+            var client = _factory.CreateClient();
+
+            var response = await client.GetAsync("system/ping").ConfigureAwait(false);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
```

Comment:
```suggestion
```

`System/Ping` doesn't belong in the `EncodedQueryStringTest` class

Reply:
Going mad - thought i'd removed it!